AWS/Lambda/PREmailer/pr_emailer.py

```diff
@@ -386,56 +386,45 @@ def lambda_handler(event, context):
     gh_pr = gh.get_repo('llvm/llvm-project').get_issue(pr_number).as_pull_request()
     for commit in gh_pr.get_commits():
         project_list.update(create_project_list([f.filename for f in commit.files], project_path_email))
-    # Track the email address last sent to
-    last_mail_to = ""
 
     # Iterate through the list of projects and cross-post if necessary
     #print(project_list)
     #print(body)
-    for project in project_list:
-        # setup the MailTO
-        # separate multiple recipient by comma. eg: "abc@gmail.com, xyz@gmail.com"
-        # mail_to = os.environ['MAIL_TO']
-        mail_to = project_path_email[project]
-
-        if not is_main_branch_event(event, gh_pr):
-            mail_to = LLVM_BRANCH_COMMITS_ADDRESS
-
-        # If we're sending an additional email to the same address, break instead
-        if mail_to == last_mail_to:
-            break
-
-        if 'MAIL_TO' in os.environ:
-            mail_to = os.environ.get('MAIL_TO')
-        # Setup the mail Subject
-        subject = f"[{project}] {pr_title} (PR #{pr_number})"
```

Comment:
This patch inadvertently dropped the project tag from the beginning of the subject, so I've rolled back to a previous version.  I'll send another PR to fix this.